File: kits/ts/myBot.ts
import { Agent } from './kit';

// create a new agent
const agent = new Agent();

// first initialize the agent, and then proceed to go in a loop waiting for updates and running the AI
agent.initialize().then(async () => {
  while(true) {
    
    // wait for update from match engine
    await agent.update();

    /** AI Code goes here */

    let commands: Array<string> = [];

    // push some commands in to be processed by the `MatchEngine` working under a `Design`
    commands.push('somecommand');
    commands.push('anothercommand');

    // submit commands to the `MatchEngine` and the `Match`, using ',' as the delimiter
    console.log(commands.join(','));
    
    // now we end our turn
    agent.endTurn();
    
  }
});

File: kits/python/myBot.py
from kit import Agent

# Create new agent
agent = Agent()

# initialize agent
agent.initialize()

# Once initialized, we enter an infinite loop
while True:

    # wait for update from match engine
    agent.update()

    commands = []

    # AI Code goes here #

    # push some commands in to be processed by the `MatchEngine` working under a `Design`
    commands.append('somecommand')
    commands.append('anothercommand')

    # submit commands to the `MatchEngine` and the `Match`, using ',' as the delimiter
    print(','.join(commands))

    # now we end our turn
    agent.end_turn()